drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1147,8 +1182,11 @@ function DragAndDropBlock(runtime, element, configuration) {
             popup_html: state.feedback || '',
             overall_feedback_messages: state.overall_feedback,
             disable_reset_button: !canReset(),
+            disable_show_answer_button: !canShowAnswer(),
             disable_submit_button: !canSubmitAttempt(),
-            submit_spinner: state.submit_spinner
+            submit_spinner: state.submit_spinner,
+            show_answer: state.show_answer,
```

Comment:
@arbrandes `state.show_answer` represents information about whether the board is currently displaying the correct answer, right? If so, I think it would make sense to rename `show_answer` to `showing_answer`. (`show_answer` can also be interpreted as "allowed to show answer", so there's some potential for confusion.)